src/pages/CatNew.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+
+class CatNew extends Component {
+  render() {
+    return (
+      <>
+        <h1>Welcome to the club, cat!</h1>
+      </>
+      );
+  }
+}
```

Comment:
This component looks perfect. I would use this syntax structure and indentation as a model for all the other components.